sequencer/sequencesender.go

```diff
@@ -152,7 +152,7 @@ func (s *Sequencer) handleEstimateGasSendSequenceErr(
 	err error,
 ) ([]types.Sequence, error) {
 	// Insufficient allowance
-	if strings.Contains(err.Error(), errInsufficientAllowance) {
+	if errors.Is(err, ethman.ErrInsufficientAllowance) {
```

Comment:
```suggestion
	if errors.Is(err, etherman.ErrInsufficientAllowance) {
```

Reply:
this is not valid as it collides with the interface inside the package: https://github.com/0xPolygonHermez/zkevm-node/blob/develop/sequencer/interfaces.go#L32